### DISCUSSION TITLE
Créditos para el Despliegue

### DISCUSSION THREAD

**Discussion by marvicmar** [Aparking/AparKing_Backend#148] [General]
# Resumen
¡Hola! He creado esta discusión para dejar una breve explicación de cómo podéis solicitar los créditos de App Engine para desplegar nuestro backend. Además, me gustaría posteriormente **enviéis** el código que recibáis por **privado** a través de nuestro dicord. Más abajo explicaré cómo seguir, ya que es necesario poder agruparlos en la cuenta donde realizamos los despliegues.

## Solicitar Créditos
En primer lugar, debereís acceder al siguiente [enlace](https://vector.my.salesforce-sites.com/GCPEDU?cid=pt%2B7hq6TLZ0hP/TAeHJcKq4g3gpKPpepPhulP6MCRuNmJLHb1Gf7lux5NWsBCI3H/) este os mostrara una pantalla como esta, la cuál debéis rellenar con vuestros datos: 
![image](https://github.com/Aparking/AparKing_Backend/assets/73558808/335a3659-8295-4428-9b52-df6e9352fabe)

## Email App Engine
Tras esto os llegara a vuestro correo de la US un mensaje como el siguiente: 
![Imagen de WhatsApp 2024-03-17 a las 19 13 58_a51830a6](https://github.com/Aparking/AparKing_Backend/assets/73558808/e1ad7eaf-e050-4f53-a7a1-28586c60193a)

### Lo que necesito es que copies el código de color naranja y me lo paséis.

**Comment by MarioArocaPaez** [comment]
De todas formas @marvicmar quizás no es muy seguro poner los códigos por aquí ya que en teoría son privados y este repo es publico. Si quieres en 5 minutos los pongo todos en un documento en nuestro drive que si es privado

**Reply by MarioArocaPaez** [reply]
También no sé como lo estáis llevando ahora mismo, si queréis lo hablamos en cualquier momento. Pero en teoría deberíamos tener 3 enlaces siempre funcionando, uno por cada Sprint
![imagen](https://github.com/Aparking/AparKing_Backend/assets/73229438/1ec49512-aade-4b1e-9985-77bcc51c30ef)

**Reply by marvicmar** [reply]
La verdad es que pensaba que era privado el reto, así que fallo por mí parte gracias por la apreciación y si estaría bien lo que comentas. 

Por otro lado respecto a los enlaces para la Landing Page, en principio ya se ha podido desplegar el frontend porque antes solo tenían el backend y no podían hacer mucho, así que me informo un poco del estado y en cuanto lo tenga te paso enlace para que puedas actualizarlo.
